src/core/eme/handle_session_events.ts

```diff
@@ -143,11 +143,19 @@ export default function handleSessionEvents(
       (session.keyStatuses as any).forEach((keyStatus : string, keyId : string) => {
         // Hack present because the order of the arguments has changed in spec
         // and is not the same between some versions of Edge and Chrome.
-        if (KEY_STATUS_ERRORS[keyId]) {
+        if (keyId === KEY_STATUS_INTERNAL_ERROR) {
           throw new EncryptedMediaError("KEY_STATUS_CHANGE_ERROR", keyId, true);
-        } else if (KEY_STATUS_ERRORS[keyStatus]) {
+        } else if (keyStatus === KEY_STATUS_INTERNAL_ERROR) {
           throw new EncryptedMediaError("KEY_STATUS_CHANGE_ERROR", keyStatus, true);
         }
+
+        if (!keySystem || keySystem.onKeyStatusesChange == null) {
```

Comment:
We might not need this part, depending if license expiration always send message with renewal requests.